src/main/steps/tasklistresponse/proceedings/documentUpload/template.njk

```diff
@@ -7,6 +7,12 @@
 
 
 {% extends "common/page.njk" %}
+
```

Comment:
Not required, when the title is constructed.

Reply:
Changed to set the title in content.ts